components/raftstore/src/store/snap.rs

```diff
@@ -1072,19 +1337,21 @@ struct SnapManagerCore {
     limiter: Limiter,
     temp_sst_id: Arc<AtomicU64>,
     encryption_key_manager: Option<Arc<DataKeyManager>>,
+    max_per_file_size: Arc<AtomicU64>,
+    enable_multi_snapshot_files: Arc<AtomicBool>,
 }
 
 /// `SnapManagerCore` trace all current processing snapshots.
 pub struct SnapManager {
     core: SnapManagerCore,
-    max_total_size: AtomicU64,
+    max_total_size: Arc<AtomicU64>,
```

Comment:
Why `Arc`? Is it a bug?

Reply:
Because the SnapshotManager can have several instances and any instance changes the max_total_size should impact all other instances.